### ISSUE TITLE
Sickchill cannot load object '/usr/local/lib/libkk.so'

### ISSUE THREAD

**Issue by FuNnYf-be** [SynoCommunity/spksrc#3728]
### Setup
_Package Name: Python 
_Package Version: v2.7.14-19

_NAS Model: DS216play
_NAS Architecture: monaco
_DSM version: 6.2.2-24922

### Expected behavior
The package Sickchill uses only this Python library. It gives this error:

> You're using the ERROR: ld.so: object '/usr/local/lib/libkk.so' from /etc/ld.so.preload cannot be preloaded (file too short): ignored. ERROR: ld.so: object '/usr/local/lib/libkk.so' from /etc/ld.so.preload cannot be preloaded (file too short): ignored. master branch. Please use 'master' unless specifically asked

### Actual behavior
The error should not appear. 

### Steps to reproduce
_1. Install Pyhton package 
_2. Install Sickchill package: [https://github.com/BenjV/SYNO-packages/blob/master/SickChill%20noarch%20V1.1.spk](url)
_3. Open Sickchill, error appaers

#### Package log
/

#### Other logs (Sickchill error log)
You're using the ERROR: ld.so: object '/usr/local/lib/libkk.so' from /etc/ld.so.preload cannot be preloaded (file too short): ignored. ERROR: ld.so: object '/usr/local/lib/libkk.so' from /etc/ld.so.preload cannot be preloaded (file too short): ignored. master branch. Please use 'master' unless specifically asked

**Comment by ymartin59** [comment]
Not official SynoCommunity package. Please discuss about it with @BenjV

**Comment by BenjV** [comment]
Has nothing to do with the SickChill package.
SickChill (fromaly known as SickRage) is a python application and this is a python problem on the monaco platform.
Python is trying to use "/us/local/lib.libkk.so" which on this platform is apparently an empty file.

**Comment by ymartin59** [comment]
Has nothing to do with Python package either. The only reference I have found to such a "libkk" library is http://www.kplusk-messtechnik.de/downloads/index.html
Looks like dependency management is complex for that project: https://github.com/SickChill/SickChill/tree/master/requirements

**Comment by BenjV** [comment]
That are all python dependancies and no binary libraries.
The package I made is an exact copy of the sickbeard-custom package of the SynoCommunity with an addition that it wil point to the new SickChill github and so counter the hijack Echel0n.

It is used by myself on two different Nasses and by many others without any problem of this kind.

The strange thing is that I cannot find a libkk.so neither in the python package and of course not in the SickChill app (that is only python code).

In the mean time the issue was resolved by changing the  /etc/ld.so/preload to prevent  to preload a libkk.so which was present but with size 0.
